<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseType</class>
 <widget class="QDialog" name="chooseType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chooseType</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请选择你要注册的用户类型：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="tBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>老师</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>学生</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>chooseType</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
